### PR TITLE
fix(Packaging): Add exec bit for packaged files on Windows

### PR DESCRIPTION
**Pull request by scadu**
Closes: #3557

### CODE REVIEW

File: lib/plugins/package/lib/packageService.js [hunk 1]
```diff
@@ -151,16 +132,9 @@ module.exports = {
...
-    if (process.platform === 'win32') {
-      const runtime = this.getRuntime(functionName.runtime);
-      if (runtime.startsWith('go')) {
-        filesToChmodPlusX.push(functionObject.handler);
-      }
-    }
 
     const filePaths = await this.resolveFilePathsFunction(functionName);
-    const artifactPath = await this.zipFiles(filePaths, zipFileName, undefined, filesToChmodPlusX);
+    const artifactPath = await this.zipFiles(filePaths, zipFileName, undefined);
```
Comment (medikoo) [inline lib/plugins/package/lib/packageService.js:137]: Let's remove `undefined` argument

**Review by medikoo — CHANGES_REQUESTED** [review]
@scadu that looks great! I have just one very minor cleanup suggestion, and it'll be ready to go

File: lib/plugins/package/lib/packageService.js (hunk 1 above)
Comment (scadu) [reply lib/plugins/package/lib/packageService.js]: Done.

**Review by medikoo — APPROVED** [review]
Thank you @scadu !

**Pull request merged**